utils/file.go

```diff
@@ -146,12 +146,20 @@ func CopySingleFile(src, dst string) (int64, error) {
 	return nBytes, err
 }
 
-func CleanFile(file *os.File) (err error) {
+func CleanFile(file *os.File) {
 	if file == nil {
 		return
 	}
-	err = file.Close()
-	return os.Remove(file.Name())
+	// the following operation won't failed regularly, if failed, log it
+	err := file.Close()
+	if err != nil {
+		logger.Warn(err)
+	}
+	err = os.Remove(file.Name())
+	if err != nil {
+		logger.Warn(err)
```

Comment:
The recommended way is to return the error, caller decision write log or not

Reply:
I suppose os.Remove/Close ... won't return err regularly. So if error occurs, we log it.